package.json

```diff
@@ -14,7 +14,7 @@
     "heroku-postbuild": "npm run build",
     "start": "node server.js"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
This change is important. When using `stateless-components` in mab (or any other project), it wanted to install all these dependencies with webpack and shrinkwrap was failing with `peer invalid: webpack@^3.1.0, required by extract-text-webpack-plugin@3.0.2`

Reply:
Hm, that doesn't happen to me. Also when using devDependencies Heroku build fails.

Reply:
Ah, sorry about that. Well, maybe because on dashboard you already have webpack 3 or something.

Reply:
How about if we change Heroku config so it will install `devDependencies` too? https://stackoverflow.com/questions/22954782/install-devdependencies-on-heroku